### DISCUSSION TITLE
How to send alerts in elastalert to some user defined api exposed in our application

### DISCUSSION THREAD

**Discussion by Thara-123** [jertel/elastalert2#767] [Q&A] [answered]
We have used elastalert to monitor logs in elasticsearch. If I configure email I get the mail and everything is working as expected.

Now the ask is to send the alerts to an API that is exposed in our application. So that this API will get the alert process it accordingly.

As far as I searched the alert can be sent to email,slack,MS teams,googlechat,telegram etc but how to send it to some userdefined URL

Any help on this will be really appreciated

Thanks in advance

**Comment by iamxeph** [comment, marked as answer]
[https://elastalert2.readthedocs.io/en/latest/ruletypes.html#http-post-2](https://elastalert2.readthedocs.io/en/latest/ruletypes.html#http-post-2)

OR you could write a new alerter [https://elastalert2.readthedocs.io/en/latest/recipes/adding_alerts.html](https://elastalert2.readthedocs.io/en/latest/recipes/adding_alerts.html)

**Reply by Thara-123** [reply]
This is what I really wanted.. .Thank you so much. This fixed my issue
